### PR TITLE
Decode staking event logs

### PR DESCRIPTION
**Pull request by y4t0g4m1**
# Type of change

- [X] Decode staking-related events into a map for notification

### CODE REVIEW

File: server/info.go [hunk 1]
```diff
@@ -636,6 +636,13 @@ func (s *infoServer) mergeAdditionalInfoToTxs(txs []*types.Transaction, receipts
 		}
 
 		tx.Logs = receipts[receiptIndex].Logs
+		for i, log := range tx.Logs {
+			result, err := s.kaiClient.UnpackLog(&log)
```
Comment (ngdlong91) [inline server/info.go:640]: 🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>G601: Implicit memory aliasing in for loop. (gosec)

File: kardia/abi.go [hunk 2]
```diff
@@ -88,6 +89,69 @@ func (ec *Client) DecodeInputData(to string, input string) (*types.FunctionCall,
...
+		}
+	}
+	// unpacking indexed arguments
+	var indexed abi.Arguments
+	for _, arg := range a.Events[eventName].Inputs {
+		if arg.Indexed {
+			indexed = append(indexed, arg)
+		}
+	}
+	var topics []common.Hash
```
Comment (ngdlong91) [inline kardia/abi.go:148]: 🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Consider preallocating `topics` (prealloc)

File: server/info.go (hunk 1 above)
Comment (y4t0g4m1) [reply server/info.go]: Resolved

File: kardia/abi.go (hunk 2 above)
Comment (y4t0g4m1) [reply kardia/abi.go]: resolved

**Pull request closed without merging**